config/rbac/role.yaml

```diff
@@ -169,6 +169,7 @@ rules:
   verbs:
   - get
   - list
+  - update
```

Comment:
hmm, which rbac needs to be updated?

I am not about  operators updating rbac. :)

Reply:
The reconcile loop has to update InferenceService CRs, adding some ad-hoc labels to keep the links between the CR and the record in model registry - that's why I had to add that `update`. Is there any other way I can do that?